src/vm/eventpipefile.cpp

```diff
@@ -3,15 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 #include "common.h"
-#include "eventpipebuffer.h"
 #include "eventpipeblock.h"
-#include "eventpipeconfiguration.h"
 #include "eventpipefile.h"
 #include "sampleprofiler.h"
 
 #ifdef FEATURE_PERFTRACING
 
-EventPipeFile::EventPipeFile(SString &outputFilePath)
+EventPipeFile::EventPipeFile(StreamWriter *pStreamWriter) : FastSerializableObject(3, 0)
```

Comment:
Can we make the `ObjectVersion` an enum defined in `FastSerializableObject`? It's extremely confusing what this 3 means as opposed to the 1 in `EventPipeBlock`.

Reply:
It makes sense, but we should revisit it in a later PR (I'm not sure if tracking version should be an `enum` neither). I just made the values `const` to enforce derived classes initializing these values on construction.